ebsco/edsapi-koha-plugin
# Encode json query

This update encodes the username and password leveraging the Perl JSON library to avoid issues with special characters in EDS API auto generated Usernames and Passwords.

To Test:

If you have an instance of Koha to install this on, install the eds_plugin_19.05008.kpz

Otherwise contact me for admin logins to the local Koha Plugin install.

Go into the plugin configuration.
Use a valid username and password that includes special characters (for example something like: th1s!ma'ybr~e@k"stuff )
Save the values
Open the configuration and verify they still appear as expected.
Click Load setting from EBSCOAdmin Now
Verify that you get the Info Stored method.
Do test searches in the EDS API Search interface to verify that the credentials are being used correctly in searches.
